packages/lexicons/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@tsky/lexicons",
+  "type": "module",
+  "version": "0.0.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
```

```suggestion
  "scripts": {
    "build": "tsx ./scripts/generate-types.ts"
  },
  "dependencies": {
```

This will allow ci to build it as well